microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -525,8 +527,26 @@ _new as of v106_
 Whether to hide or display code that's in the ignore list, in the **Sources** tool's tree view, such as the **Page** tab.
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary v106.
-*  This checkbox is not present in Microsoft Edge Stable v104.
+*  This checkbox is present in Microsoft Edge Canary v107.
+*  This checkbox is not present in Microsoft Edge Stable v105.
+
+
+<!-- ====================================================================== -->
+## Highlight important DOM properties in the Object Properties viewer
+
+_new as of v107_
```

Comment:
@captainbrosset - need text.  What tool contains the Object Property viewer?

Reply:
The Object Property viewer can be found in several tools: Elements, Sources, Console, Network, maybe others. It's a shared component that DevTools uses in several places to provide a treeview user interface that lets users expand properties of objects, and sub-properties, etc.

You can see it easily in the Elements tool:
1. Navigate to any page.
2. Select any element in the page in the Elements tool.
3. In the sidebar, select the **Properties** tab.

What you see in the tab is the list of properties for the selected element, and it is rendered by the Object Properties viewer.

This new experiment makes certain important properties easier to see. Some appear in bold, and others appear in bold **and** have a star icon next to them.

The goal of this experiment is to make these properties easier to see. This is useful because in JavaScript, objects inherit from parent objects, which themselves inherit from other ancestors, and so on. So, you end up very quickly with hundreds of properties on any given object, making it hard to find the ones that you care about.

Reply:
incorp'd info